src/ddmd/objc.d

```diff
@@ -117,126 +117,140 @@ struct ObjcSelector
     }
 }
 
-struct Objc_ClassDeclaration
-{
-    // true if this is an Objective-C class/interface
-    bool objc;
+private __gshared Objc _objc;
 
-    // MARK: Objc_ClassDeclaration
-    extern (C++) bool isInterface()
-    {
-        return objc;
-    }
+Objc objc()
+{
+    return _objc;
 }
 
-struct Objc_FuncDeclaration
+interface Objc
```

Comment:
@ibuclaw The change to `extern(C++)` seems to cause every platform to fail in the auto tester. But it works fine locally (on macOS).